ql/src/java/org/apache/hadoop/hive/ql/exec/MemoryInfo.java

```diff
@@ -46,30 +48,25 @@ public MemoryInfo(Configuration conf) {
       llapInfo.initClusterInfo();
       if (llapInfo.hasClusterInfo()) {
         this.maxExecutorMemory = llapInfo.getMemoryPerExecutor();
+        LOG.info("Using LLAP registry executor MB {}", (maxExecutorMemory / 1024L * 1024L));
```

Comment:
I think there's a type here, maxExecutorMemory I believe is already is already in MB?  Regardless, there's a typo here... I think you want to multiple 1024 twice, not divide :)

Reply:
maxExecutorMemory is in bytes (see how we handle it on Tez and MR mode) -- however LlapCluster state does the conversion prematurely as part of: https://github.com/apache/hive/blob/5eebbdf7c5750b31e1c43fe576fc0ab728bce05c/ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/LlapClusterStateForCompile.java#L147

Reply:
Thanks, however `/ 1024L * 1024L` is a no-op

Reply:
Good catch, thanks!